RELEASE-NOTES.md

```diff
@@ -1,4 +1,8 @@
 ## RELEASE NOTES
+### Version 4.0.0-EUI-3528-4205-enhancements
+**EUI-3528** "Retain Hidden Value" support for DynamicList fields, _only_ for `retain_hidden_value` flag set to `false` or unspecified
+**EUI-4205** "Retain Hidden Value" - prevent deletion of Complex sub-field value if parent unknown
+
```

Comment:
Hi @rayliangatos - it might yet be included in Release `4.1.0`, depending on @Mo-Lala84. So, for now, I've just made it a tag release aimed at `4.0.0` but it can be updated once the final version is decided.

Reply:
@rayliangatos - is it OK to resolve this conversation?